tools/Environments/DevHome.Environments/ViewModels/ComputeSystemCardBase.cs

```diff
@@ -59,18 +60,29 @@ public abstract partial class ComputeSystemCardBase : ObservableObject
 
     public string ComputeSystemId { get; protected set; } = string.Empty;
 
+    private readonly StringResource _stringResource = new("DevHome.Environments.pri", "DevHome.Environments/Resources");
+
+    private readonly StringResource _stringResourceCommon = new("DevHome.Common.pri", "DevHome.Common/Resources");
+
     [ObservableProperty]
     private string _uiMessageToDisplay = string.Empty;
 
     public ComputeSystemCardBase()
     {
-        var stringResource = new StringResource("DevHome.Environments.pri", "DevHome.Environments/Resources");
-        _moreOptionsButtonName = stringResource.GetLocalized("MoreOptionsButtonName");
+        _moreOptionsButtonName = _stringResource.GetLocalized("MoreOptionsButtonName");
     }
 
     public override string ToString()
     {
-        return $"{Name} {AlternativeName}";
+        StringBuilder description = new(Name);

```

Comment:
NIT change to `var description = new StringBuilder(Name);` to stay consistent with the rest of DevHome.

Reply:
I have been thinking about the 'right' way. Makes sense. Will change.